test/indexing.jl

```diff
@@ -695,7 +759,7 @@ end
     @test sdf[!, :x2] isa SubArray
     @test parent(sdf[!, :x2]) === df.x2
     @test_throws ArgumentError @view sdf[!, :x]
-    @test_throws ArgumentError @view sdf[!, 1:2]
+    @test_throws select(sdf, 1:2, copycols=false) == @view sdf[!, 1:2]
```

Comment:
?

Reply:
This is a remainder of changed rules. This is now allowed (since yesterday 😄) and I will fix it. Thank you!